src/electron/main.ts

```diff
@@ -91,7 +91,10 @@ const userStore = new ElectronStore();
 					const result = await Persistence.read<Types.SavedProject>(pathToOpen);
 
 					if (result.state === PersistenceState.Error) {
-						// TODO: Show user facing error here
+						dialog.showErrorBox(
+							'We ran into a problem!',
+							'Sorry, we had trouble while opening that file. It may help to update your version of Alva.'
```

Comment:
Let's change to `Sorry, we had trouble while opening the file ${pathToOpen}`. I'd discard the second part of the message because we'll be capable of detecting version mismatches in the future reliably (see #447)